Tests/ShamTests/Tests/StubRequestTests.swift

```diff
@@ -6,10 +6,24 @@ import UtilityBeltNetworking
 import XCTest
 
 final class StubRequestTests: XCTestCase {
-    func testRandomizedQueryStringOrderComparisonSucceeds() {
-        let request: StubRequest = .get("https://spothero.local?shape=triangle&color=blue")
-        let canMockData = request.canMockData(for: .get("https://spothero.local?color=blue&shape=triangle"))
+    
+    private let baseURLString = "https://spothero.local"
+    
+    func testDescriptionWithUnsortedParameters() {
+        let unsortedQueryParametersURL = "\(self.baseURLString)?zebra=thing&ardvark=other_thing"
```

Comment:
Hate to do this to you in a test, but it's `aardvark` with two `a`'s. 😐

```suggestion
        let unsortedQueryParametersURL = "\(self.baseURLString)?zebra=thing&aardvark=other_thing"
```

Reply:
knew i shouldve just went with "ant"....